e2e/README.md

```diff
@@ -0,0 +1,121 @@
+# E2E performance regression tests
+
+This directory contains the scripts and configuration files for running the
+performance regression tests. These tests are called E2E tests because they
+run the app on a real device (physical or emulated).
+
+![Example of a e2e test run](./example.gif)
```

Comment:
yes, I will upload that gif somewhere else, or just remove it if we feel it adds no value. However we need to commit the e2e dir to the repository, otherwise, we would be missing the code for running the tests.